cluster/addons/dns/kube-dns/kube-dns.yaml.base

```diff
@@ -84,6 +84,9 @@ spec:
       annotations:
         scheduler.alpha.kubernetes.io/critical-pod: ''
         seccomp.security.alpha.kubernetes.io/pod: 'docker/default'
+        prometheus.io/path: /metrics
```

Comment:
Isn't `metrics` the default so we can omit this annotation?

Reply:
fixed!